test/test_type_hints.py

```diff
@@ -170,6 +170,20 @@ def test_doc_examples(self):
             except subprocess.CalledProcessError as e:
                 raise AssertionError("mypy failed.  Look above this error for mypy's output.")
 
+    def test_type_hint_examples(self):
+        """
+        Runs mypy over all the test examples present in
+        type_hint_tests directory
+        """
+
+    examples = os.listdir(os.getcwd() + '/test/type_hint_tests/')
+    for i in examples:
+        process = subprocess.Popen(['mypy', os.getcwd()
+                                   + '/test/type_hint_tests/' + i],
```

Comment:
Using "/" is platform-dependent. For example, windows prefers "\". For that reason it's better to use os.join to combine paths

Reply:
Fixed